flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/catalog/CatalogManagerTest.java

```diff
@@ -589,6 +600,7 @@ void testCatalogStore() throws Exception {
         assertTrue(catalogManager.listCatalogs().contains("cat2"));
         assertTrue(catalogManager.listCatalogs().contains("cat3"));
         assertTrue(catalogManager.listCatalogs().contains("cat_comment"));
+        assertFalse(catalogManager.listCatalogs().contains("cat_no_type"));
```

Comment:
Would it be better to put this assertion together with the above assertThatThrownBy?